edgekv/README.md

```diff
@@ -0,0 +1,49 @@
+# Database Schema
+## Hint entry file
+**Name**|**Size**|**Rust Type**|**Contents**
+:-----:|:-----:|:-----:|-----
+timestamp|8| `i64`| Timestamp in utc encoded to big-endian bytes
+key_size| 8| `u64`| Size of the key encoded to big-endian bytes
+value_size|8| `u64`| Size of the value encoded to big-endian bytes
+data_entry_position| 8| `u64`| Position of the beginning of data in `Data file`
+key|variable| `Vec<u8>`| Key
+
+
+Hint entry is appended to the `{file_id}.hint` with this encoding structure
+
+```
++-----------+----------+------------+---------------------+-----+
+| timestamp | key_size | value_size | data_entry_position | key |
++-----------+----------+------------+---------------------+-----+
+```
+
+
+## Data entry file
+| **Name**    | **Size**   | **Rust Type**  | **Contents**            |
+|:-----------:|:----------:|:--------------:|-------------------------|
+| crc         | 4          | `u32`          | cyclic redundancy check |
+| timestamp   | 8          | `i64`          | Timestamp               |
+| key_size    | 8          | `u64`          | Key size                |
```

Comment:
@mambisi do we need keys and values that big? (but specially for keys that range seems excessive, shouldn't 2 bytes be plenty? allows for keys of up to 65536 in length).